Cargo.toml

```diff
@@ -97,6 +97,7 @@ license = "MIT"
 
 [workspace.dependencies]
 aes = "0.8.2"
+ahash = "0.8.3"
```

Comment:
Shouldn't this be marked as optional?

Reply:
No idea how workspace dependencies work, should it?

Reply:
Ah, I just looked and we don't do this for all of the adjacent dependencies, so this is probably fine.